lib/middleware.ts

```diff
@@ -68,6 +69,22 @@ function getIp(req) {
   return req.ip || remoteAddress;
 }
 
+/**
+ * Log level
+ */
+function getLogLevel(req, res) {
+  const statusCode = res.statusCode;
+  let result;
+  if (statusCode >= 500) {
+    result = kayveeLogger.Error;
+  } else if (statusCode >= 400) {
```

Comment:
Any estimates on how log level currently corresponds to status code? If there are some standard practices that make sense, can we keep them for consistency?